src/Umbraco.Cms.Integrations.Crm.Dynamics/Models/Dtos/IdentityDto.cs

```diff
@@ -4,12 +4,14 @@ namespace Umbraco.Cms.Integrations.Crm.Dynamics.Models.Dtos
 {
     public class IdentityDto
     {
-        public bool IsAuthorized { get; set; } = true;
+        public bool IsAuthorized { get; set; }
```

Comment:
Are you missing a `[JsonProperty("isAuthorized")]` here?

Reply:
This flag is only used server-side, for the client I am using the _IsAuthorized_ serializable property of _OAuthConfigurationDto_ .